api/base/serializers.py

```diff
@@ -653,7 +676,8 @@ def to_representation(self, obj, envelope='data'):
                         data['embeds'][field.field_name] = result
                 else:
                     try:
-                        data['relationships'][field.field_name] = field.to_representation(attribute)
+                        if not (is_anonymous and hasattr(field, 'view_name') and field.view_name.find('users:') != -1):
```

Comment:
For sure, a mixin would work. Just some thing that a pycharm refactor by an intern probably won't break.

If you're not worried about it, I'm not. You'll probably be the one refactoring it anyway whenever it changes.